tests/multiple-tests-single-fail/expected_results.json

```diff
@@ -2,37 +2,49 @@
   "status" : "fail",
   "tests" : [
     {
-      "message" : "/solution/Tests/MultipleSingleFailTests/MultipleSingleFailTests.swift:9: error: MultipleSingleFailTests.testAdd : XCTAssertEqual failed: (\"0\") is not equal to (\"5\") - 2+3 should equal 5",
-      "name" : "MultipleSingleFailTests.testAdd",
-      "output" : "yabba…",
-      "status" : "fail"
+      "message" : "(\"0\") is not equal to (\"5\") - 2+3 should equal 5",
+      "name" : "testAdd",
+      "output" : "yabba",
+      "status" : "fail",
+      "task_id" : 1,
```

Comment:
It looks like all exercises now have a `task_id`, which is not how we envisioned it. Couldn't you have a contention where the class needs to start with `Task` or `Task1` or something, to distinguish between concept exercise tests and practice exercise tests?

Reply:
I have thought that I may compare the first and the last test and if they have the same task id I remove them? Since even if there is a concept with only 1 task it still aint that useful.

Reply:
> I have thought that I may compare the first and the last test and if they have the same task id I remove them? Since even if there is a concept with only 1 task it still aint that useful.

Well, it does change how the UI is rendered, so I'd like it to be there even then.